packages/website/src/examples/layout/Header/Header.demo.tsx

```diff
@@ -9,9 +9,15 @@ const Demo = () => (
         </Header.Breadcrumbs>
         Title
         <Header.DocAnchor href="https://about:blank" label="Tooltip text" />
-        <Header.Actions>
-            <Button>Primary</Button>
-            <Button variant="outline">Secondary</Button>
+        <Header.Actions wrap="nowrap">
+            <Stack gap="sm" bg="gray.3" p="sm">
+                <Text>Some content</Text>
+                <Text>Some more content</Text>
+            </Stack>
+            <Group gap="sm">
+                <Button>Primary</Button>
+                <Button variant="outline">Secondary</Button>
+            </Group>
```

Comment:
I have the feeling this is a pretty specific use case, the most common happy path is to have only buttons in the header's right section

```suggestion
            <Button>Primary</Button>
            <Button variant="outline">Secondary</Button>
```